module/VuFind/src/VuFind/View/Helper/Root/SearchMemory.php

```diff
@@ -48,14 +50,22 @@ class SearchMemory extends AbstractHelper
      */
     protected $memory;
 
+    /**
+     * Search table
+     *
+     * @var Search
+     */
+    protected $searchTable;
+
     /**
      * Constructor
      *
      * @param Memory $memory Search memory
      */
-    public function __construct(Memory $memory)
+    public function __construct(Memory $memory, Search $search)
     {
         $this->memory = $memory;
+        $this->searchTable = $search;
```

Comment:
I see that you removed some of the unused parameters, but searchTable also still seems to be unused.

Reply:
Fixed.